src/cdk/testing/selenium-webdriver/selenium-web-driver-element.ts

```diff
@@ -154,6 +154,19 @@ export class SeleniumWebDriverElement implements TestElement {
     );
   }
 
+  /**
+   * Sets the text of the element to the specified value.
+   * @param text Text to be set on the element.
+   */
+  async setText(text: string): Promise<void> {
+    await this._stabilize();
+    return this._executeScript(
+      (element: Element, textToSet: string) => (element.textContent = textToSet),
```

Comment:
In the spirit of keeping harnesses behaving like a user would, I think we should add a check here `if (element.cotentEditable)`, maybe throw an error in the `else` case?

Reply:
This isn't directly on a harness though, it's on the test element. I'm not sure that we can keep this behavior up, considering that we also support dispatching arbitrary custom events.